resources/Datastore_test.go

```diff
@@ -17,7 +17,7 @@ var _ = ginkgo.Describe("Datastore", func() {
 		err       error
 	)
 
-	ginkgo.Describe("getters", func() {
+	ginkgo.Describe("test getters and datastore is created from XML", func() {
```

Comment:
Usually if the names of multiple `Describe` blocks starts with the same string it should be divided into `Context` blocks under that `Describe` block :wink: